plugins/entity-feedback/src/components/FeedbackResponseDialog/FeedbackResponseDialog.tsx

```diff
@@ -130,19 +131,31 @@ export const FeedbackResponseDialog = (props: FeedbackResponseDialogProps) => {
               />
             ))}
           </FormGroup>
+          {Object.keys(responseSelections).every(
+            key => responseSelections[key] === false,
+          ) ? (
+            <FormHelperText error>
```

Comment:
As I mentioned in my [comment](https://github.com/backstage/backstage/issues/18661#issuecomment-1634205334) in the issue, I think it would be preferred to have validation be an optional feature (default to false to preserve existing behavior), perhaps controlled via prop to this component